Keen.NetStandard/Query/QueryDefinition.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+
+
+namespace Keen.Core.Query
+{
+    /// <summary>
+    /// Holds information describing the query that is cached within a cached dataset.
+    /// </summary>
+    public class QueryDefinition
```

Comment:
I'd like to see this eventually become how we generally define a query and pass parameters to and around within the SDK.

Reply:
Agree

Reply:
Especially as a part of fixing Issues #32 and #42 my plan has been to do this since once we move to POSTing all queries it'll be cleaner.